mvpa2/algorithms/searchlight_hyperalignment.py

```diff
@@ -262,9 +263,10 @@ class SearchlightHyperalignment(ClassWithCollections):
             If None, hyperalignment is performed at every voxel (default).""")
 
     hyperalignment = Parameter(
-        Hyperalignment(ref_ds=0),
+        Hyperalignment(ref_ds=None),
         doc="""Hyperalignment instance to be used in each searchlight sphere.
             Default is just the Hyperalignment instance with default parameters.
+            Its `ref_ds` parameter would be overridden and therefore has no effect.
```

Comment:
On Tue, 23 Aug 2016, Feilong Ma wrote:

>    In mvpa2/algorithms/searchlight_hyperalignment.py:

>  >          doc="""Hyperalignment instance to be used in each searchlight sphere.
>  >              Default is just the Hyperalignment instance with default parameters.
>  > +            Its `ref_ds` parameter would be overridden and therefore has no effect.

>    Essentially we just need one ref_ds for everything.

>    We want to use the same ref_ds for both the SearchlightHyperalignment
>    instance and the Hyperalignment instance it's using for hyperaligning
>    local voxels/nodes.

>    Therefore we just set the ref_ds parameter of Hyperalignment with the
>    ref_ds parameter of SearchlightHyperalignment while running, no matter
>    what that parameter was.

so say so explicitly here please ;) ("will be overridden with ref_ds of
the SearchlightHyperalignment instance" or alike)